.github/workflows/android.yml

```diff
@@ -0,0 +1,26 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+        cache: gradle
+
+    - name: Grant execute permission for gradlew
```

Comment:
Is this needed? We aren't doing it in other actions (e.g. https://github.com/googlecodelabs/android-dagger/blob/main/.github/workflows/build_test.yml)